v21.1/cockroach-userfile-upload.md

```diff
@@ -125,6 +125,126 @@ cockroach userfile upload /Users/maxroach/Desktop/test-data.csv userfile://testd
+{{site.data.alerts.end}}
+
+After connecting to the cluster that contains the data you would like to backup, run the [`BACKUP`](backup.html) statement:
+
+~~~sql
+BACKUP DATABASE movr TO 'userfile://defaultdb.public.userfiles_$user/database-backup' AS OF SYSTEM TIME '-10s';
+~~~
+
+This will make a backup of the database under `database-backup` in the default `userfile` space.
+
+~~~
+job_id             |  status   | fraction_completed | rows | index_entries | bytes
+-------------------+-----------+--------------------+------+---------------+---------
+679645548255936513 | succeeded |                  1 | 2997 |          1060 | 492713
+(1 row)
+~~~
+
+Next, on the command line, use the following [`cockroach userfile get`](cockroach-userfile-get.html) command to fetch the files stored in the `userfile` storage and download them to your local directory:
+
+~~~shell
+cockroach userfile get 'userfile://defaultdb.public.userfiles_$user/database-backup' --url {CONNECTION STRING}
+~~~
+
+Note that without passing a specific directory this command will fetch all files stored within the default user space in `userfile`.
+
+The output will show the files downloading:
+
+~~~
+downloaded database-backup/BACKUP-CHECKPOINT-679645548255936513-CHECKSUM to database-backup/BACKUP-CHECKPOINT-679645548255936513-CHECKSUM (4 B)
+downloaded database-backup/BACKUP-CHECKPOINT-CHECKSUM to database-backup/BACKUP-CHECKPOINT-CHECKSUM (4 B)
+downloaded database-backup/BACKUP-STATISTICS to database-backup/BACKUP-STATISTICS (53 KiB)
+downloaded database-backup/BACKUP_MANIFEST to database-backup/BACKUP_MANIFEST (2.6 KiB)
+downloaded database-backup/BACKUP_MANIFEST-CHECKSUM to database-backup/BACKUP_MANIFEST-CHECKSUM (4 B)
+downloaded database-backup/data/679645557047099393.sst to database-backup/data/679645557047099393.sst (5.8 KiB)
+. . .
+downloaded database-backup/data/679645558404448257.sst to database-backup/data/679645558404448257.sst (5.8 KiB)
+downloaded database-backup/data/679645558408249345.sst to database-backup/data/679645558408249345.sst (41 KiB)
+~~~
+
+At this point, you have backed up the database to `userfile` storage and you have a copy of that backup on your local machine.
+
+Your backup will contain files similar to the following structure:
+
+~~~
+database-backup/
+├── BACKUP-CHECKPOINT-679645548255936513-CHECKSUM
+├── BACKUP-CHECKPOINT-CHECKSUM
+├── BACKUP-STATISTICS
+├── BACKUP_MANIFEST
+├── BACKUP_MANIFEST-CHECKSUM
+└── data
+    ├── 679645557047099393.sst
+    ├── 679645557048475649.sst
+    ├── 679645557048868865.sst
+    ├── 679645558151741442.sst
+. . .
+~~~
+
+If you should need to recover that data, or, apply it for testing purposes, you can now upload these files to a cluster with `cockroach userfile upload` and [`RESTORE`](restore.html).
```

Comment:
This comment may be unnecessary / a bit flow-breaking since we are doing this below